app/assets/stylesheets/pages/_home.scss

```diff
@@ -0,0 +1,47 @@
+#hero {
+  $medium-gray: #999 !default;
+  $dark-purple: #0E0866 !default;
+  $base-border-radius: 3px !default;
+  $action-color: #477DCA !default;
+  $large-screen: em(860) !default;
+  $hero-color: $medium-gray;
+
+  background: white no-repeat top;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.